dt-people-groups/people-groups-base.php

```diff
@@ -25,49 +41,564 @@ public function after_setup_theme(){
+        if ( isset( $expected_roles['administrator'] ) ) {
+            $expected_roles['administrator']['permissions'][ 'view_any_' . $this->post_type ]   = true;
+            $expected_roles['administrator']['permissions'][ 'update_any_' . $this->post_type ] = true;
+            $expected_roles['administrator']['permissions']['edit_peoplegroups']                = true;
         }
-        if ( isset( $expected_roles['dt_admin'] ) ){
-            $expected_roles['dt_admin']['permissions']['view_any_'.$this->post_type ] = true;
-            $expected_roles['dt_admin']['permissions']['update_any_'.$this->post_type ] = true;
-            $expected_roles['dt_admin']['permissions']['edit_peoplegroups'] = true;
+        if ( isset( $expected_roles['dt_admin'] ) ) {
+            $expected_roles['dt_admin']['permissions'][ 'view_any_' . $this->post_type ]   = true;
+            $expected_roles['dt_admin']['permissions'][ 'update_any_' . $this->post_type ] = true;
+            $expected_roles['dt_admin']['permissions']['edit_peoplegroups']                = true;
         }
+
         return $expected_roles;
     }
 
-    public function dt_custom_fields_settings( $settings, $post_type ){
-        return $settings;
+    public function dt_custom_fields_settings( $fields, $post_type ) {
+        if ( $post_type === $this->post_type ) {
+            /**
+             * Basic Framework Fields
+             *
+             */
+            $fields['tags']               = [
+                'name'           => __( 'Tags', 'disciple_tools' ),
+                'description'    => _x( 'A useful way to group related items.', 'Optional Documentation', 'disciple_tools' ),
+                'type'           => 'tags',
+                'default'        => [],
+                'tile'           => 'other',
+                'custom_display' => true,
+                'icon'           => get_template_directory_uri() . "/dt-assets/images/tag.svg",
+            ];
+            $fields["follow"]             = [
+                'name'    => __( 'Follow', 'disciple_tools' ),
+                'type'    => 'multi_select',
+                'default' => [],
+                'section' => 'misc',
+                'hidden'  => true
+            ];
+            $fields["unfollow"]           = [
+                'name'    => __( 'Un-Follow', 'disciple_tools' ),
+                'type'    => 'multi_select',
+                'default' => [],
+                'hidden'  => true
+            ];
+            $fields['tasks']              = [
+                'name' => __( 'Tasks', 'disciple_tools' ),
+                'type' => 'post_user_meta',
+            ];
+            $fields["duplicate_data"]     = [
+                "name"    => 'Duplicates', //system string does not need translation
+                'type'    => 'array',
+                'default' => [],
+            ];
+            $fields['assigned_to']        = [
```

Comment:
tags, follow, unfollow, tasks are default fields. No longer needed to be declared.
Duplicate data can be removed